lib/ansible/module_utils/opennebula.py

```diff
@@ -0,0 +1,333 @@
+#
+# Copyright 2018 www.privaz.io Valletech AB
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Code in module_utils must be bsd licensed.

Reply:
OK, have changed to BSD.